plugin/build_baron.go

```diff
@@ -177,7 +180,35 @@ func BbGetConfig(settings *evergreen.Settings) map[string]evergreen.BuildBaronPr
 	return projects
 }
 
-func BbGetProject(settings *evergreen.Settings, projectId string) (evergreen.BuildBaronProject, bool) {
+// BbGetProject build baron settings will be retrieved from either project or admin config depending on PluginAdminPageDisabled flag
```

Comment:
Thanks for writing a function comment! A couple of thoughts: 
Shouldn't rely on line breaks to indicate the end of a sentence, since some comments truly are just long; should use periods.
Can you try to shorten this comment up? It's not easy to skim / understand quickly what's going on. For example 186 can be simplified to "returns build baron settings and ok if found". 185 can probably be included in 184 ("project parser config takes precedence (found using version ID if given)")